docs/reference/items.md

```diff
@@ -102,66 +127,215 @@ query {
 }
 ```
 
----
+</template>
+<template #sdk>
+
+```js
+import { createDirectus } from '@directus/sdk';
+import { rest, readItems } from '@directus/sdk/rest';
+
+const client = createDirectus('https://directus.example.com').with(rest());
+
+const result = await client.request(
+	readItems('posts', {
+		fields: ['*'],
+	})
+);
+
+console.log(result);
+```
+
+</template>
+
+</SnippetToggler>
 
 ## Get Item by ID
 
 Get an item that exists in Directus.
 
-### Query Parameters
+### Request
 
-Supports all [global query parameters](/reference/query).
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+<template #rest>
 
-### Returns
+`GET /items/:collection/:id`
 
-Returns an [item object](#the-item-object) if a valid primary key was provided.
+</template>
+<template #graphql>
 
-### REST API
+`POST /graphql`
 
+```graphql
+type Query {
+	<collection>_by_id(id: ID!): <collection>
+}
 ```
-GET /items/:collection/:id
+
+</template>
+<template #sdk>
+
+```js
+import { createDirectus } from '@directus/sdk';
+import { rest, readItem } from '@directus/sdk/rest';
+
+const client = createDirectus('directus_project_url').with(rest());
+
+const result = await client.request(readItem('collection_name', 'item_id'));
+
+console.log(result);
 ```
 
-##### Example
+</template>
+
+</SnippetToggler>
+
+#### Query Parameters
+
+Supports all [global query parameters](/reference/query).
+
+### Response
+
+Returns an [item object](#the-item-object) if a valid primary key was provided.
+
+### Example
+
+<SnippetToggler :choices="['REST', 'GraphQL', 'SDK']" label="API">
+<template #rest>
+
+`GET /items/articles/15`
 
+</template>
+<template #graphql>
+
+`POST /graphql`
+
+```graphql
+type Query {
+	<collection>_by_id(id: ID!): <collection>
+}
 ```
-GET /items/articles/15
+
+</template>
+<template #sdk>
+
+```js
+import { createDirectus } from '@directus/sdk';
+import { rest, readItem } from '@directus/sdk/rest';
+
+const client = createDirectus('https://directus.example.com').with(rest());
+
+const result = await client.request(readItem('articles', '1'));
+
+console.log(result);
 ```
 
-### GraphQL
+</template>
+
+</SnippetToggler>
+
+## Get Singleton
 
+List the singleton item in Directus.
+
+### Request
+
+```js
+import { createDirectus } from '@directus/sdk';
+import { rest, readSingleton } from '@directus/sdk/rest';
+const client = createDirectus('directus_project_url').with(rest())
+
+const result = await client.request(readSingleton('collection_name'))
+
+console.log(result);
 ```
-POST /graphql
+
+::: tip Info
+
+This endpoint is only supported by the SDK.
+
+:::
```

Comment:
```suggestion
::: tip Info

This is only intended for singleton collections.

:::
```
This is not a separate endpoint but rather the same as "Get Items" (or "Get Item") endpoint for a `singleton` collection.

Reply:
Perhaps this should even be included in the example for either of the get items endpoints.

Reply:
Addressed in https://github.com/directus/directus/pull/19110/commits/04e8583c3957751ff0d273f755811210de9aa7d1.